test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestFederation.java

```diff
@@ -287,17 +292,56 @@ public void testOpensearchToCswSourceToCswEndpointQuerywithMetacardXml() throws
 
 
     @Test
-    public void testFanout() {
+    public void testFanoutQueryAgainstUnknownSource() {
         try {
-            setFanout();
+            setFanout(true);
+            waitForAllBundles();
+        } catch (IOException e) {
```

Comment:
@mverret you can just do (IOException | InterruptedException e) to make it a little cleaner